login/routers/controllers/user_controllers/UserRegistrationServices.js

```diff
@@ -0,0 +1,22 @@
+const sequelize = require('sequelize');
+
+const sequelizeOperators = sequelize.Op;
+const userModel = require('../../../db/models/user_models/UserModel.js');
+const createUserAccount = require('../../../db/user_db/CreateUserAccount.js');
+
+function signUp(registrationRequest, response) {
+  userModel.user.findOne({
+    raw: true,
+    where: {
+      [sequelizeOperators.or]: [{ login: registrationRequest.login }, { email: registrationRequest.email }],
+    },
+  })
+    .then((User) => {
+      if (!User) {
+        createUserAccount.createUserAccount(registrationRequest, response);
+        return;
+      }
+      response.status(400).send('Такой пользователь уже есть');
```

Comment:
I think this is still 409, change the method to put, since you're creating hte account.